### PR TITLE
5830 - Tooltip Puppeteer Test

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR will add puppeteer test for tooltip component

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/5830

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Run in terminal `npm run e2e:puppeteer`

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Review by yohannahbautista — DISMISSED** [review]

File: test/components/tooltip/tooltip.puppeteer-spec.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+    it('should show the title', async () => {
+      await expect(page.title()).resolves.toMatch('IDS Enterprise');
+    });
+
+    it('should show the tooltip on mouse hover', async () => {
+      const button = await page.$('#tooltip-btn');
+      button.hover();
+
+      await page.waitForTimeout(100);
```
Comment (ericangeles) [inline test/components/tooltip/tooltip.puppeteer-spec.js:17]: I think there's a way to work this without using `waitForTimeout`. Try to use `.then()` method to the `waitForSelector()`.  Something like this,

```javascript
await page waitForSelector('#element', { visible: true  }).then(...)
```

Comment (ericangeles) [reply test/components/tooltip/tooltip.puppeteer-spec.js:17]: @yohannahbautista and I have implemented this kind of approach in the other puppeteer tests. Let us know if you need help.

**Review by ericangeles — CHANGES_REQUESTED** [review]

File: test/components/tooltip/tooltip.puppeteer-spec.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+describe('Tooltip Puppeteer Tests', () => {
+  const baseUrl = 'http://localhost:4000/components/tooltip/';
```
Comment (ericangeles) [inline test/components/tooltip/tooltip.puppeteer-spec.js:2]: You have an extra `/` that causes the tests to fail.

File: test/components/tooltip/tooltip.puppeteer-spec.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+describe('Tooltip Puppeteer Tests', () => {
+  const baseUrl = 'http://localhost:4000/components/tooltip';
+  describe('Tooltips Index Page Tests', () => {
+    const url = `${baseUrl}/tooltip/example-index?theme=classic&layout=nofrills`;
```
Comment (ericangeles) [inline test/components/tooltip/tooltip.puppeteer-spec.js:4]: Remove also the `tooltip` since the baseUrl have this in the path.

**Review by ericangeles — APPROVED** [review]
Great work! 🍌

**Review by yohannahbautista — APPROVED** [review]

**Pull request merged**